.travis.yml

```diff
@@ -7,6 +7,16 @@ install:
 
 before_script:
   - npm run installWeb3
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - npm i -g npm@^3
+  - sleep 3
```

Comment:
Why is this necessary?

Reply:
@dennis00010011b could you clarify why it is necessary?

Reply:
Xvfb is needed for performs graphical operations in virtual memory. e2e script uses Metamask extension but  headless Chrome don't support extension. It is why Xvfb used

Reply:
I was asking about the `sleep 3` line.

Reply:
It is recommended code from https://docs.travis-ci.com/user/gui-and-headless-browsers/
``` sleep 3 ```  give xvfb some time to start

Reply:
@dennis00010011b did you try other setups from recommended in the link above? if yes, do they work? just curious.

Reply:
Not too much. I just tried to play with screen resolution but it wasn't helpfull

Reply:
Line 12 ```- npm i -g npm@^3``` should be excluded